Webpack Update broke `make watch`

Since commit 1953a63 `make watch` throws the following error:

```
$ make watch
[...]
node node_modules/.bin/webpack-dev-server --watch --hot --inline --port 3000 --public localcloud.icewind.me:444 --config webpack.dev.config.js
node:internal/modules/cjs/loader:928
  throw err;
  ^

Error: Cannot find module 'webpack-cli/bin/config-yargs'
Require stack:
- /home/frieder/work/nextcloud/apps-extra/groupfolders/node_modules/webpack-dev-server/bin/webpack-dev-server.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:925:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:997:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/home/frieder/work/nextcloud/apps-extra/groupfolders/node_modules/webpack-dev-server/bin/webpack-dev-server.js:65:1)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/frieder/work/nextcloud/apps-extra/groupfolders/node_modules/webpack-dev-server/bin/webpack-dev-server.js'
  ]
}
make: *** [Makefile:31: watch] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ build/main.js: node_modules $(jssources)
 
 .PHONY: watch
 watch: node_modules
-	node node_modules/.bin/webpack-dev-server --watch --hot --inline --port 3000 --public localcloud.icewind.me:444 --config webpack.dev.config.js
+	$(webpack) serve --hot --port 3000 --public localcloud.icewind.me:444 --config webpack.dev.config.js
 
 release: appstore create-tag
 

--- a/webpack.dev.config.js
+++ b/webpack.dev.config.js
@@ -65,6 +65,7 @@ module.exports = {
 		]
 	},
 	devServer: {
-		contentBase: path.resolve(__dirname, './src')
+		contentBase: path.resolve(__dirname, './src'),
+		watchContentBase: true
 	},
 };

